server/inspect.go

```diff
@@ -105,9 +106,11 @@ func (s *Server) getContainerInfo(ctx context.Context, id string, getContainerFu
 		Annotations:     ctr.Annotations(),
 		CrioAnnotations: ctr.CrioAnnotations(),
 		Root:            ctr.MountPoint(),
-		LogPath:         ctr.LogPath(),
-		Sandbox:         ctr.Sandbox(),
-		IPs:             sb.IPs(),
+		// TODO: This should probably be something to do with image store
+		// Have to look into oci.container
```

Comment:
is this TODO still relevant?

Reply:
Pushed up a fix to drop this.  Thanks for catching that.